rsts/index.rst

```diff
@@ -3,41 +3,42 @@ Flyte
 
 .. image:: images/flyte_lockup_gradient_on_light.png
 
-Flyte is a structured programming and distributed processing platform created at Lyft that enables highly concurrent, scalable and maintainable workflows for machine learning and data processing.
-Welcome to the documentation hub for Flyte.
-
-.. toctree::
-   :maxdepth: 1
-   :caption: Getting Started
-   :name: gettingstartedtoc
-
-   getting_started/first_run
-   getting_started/first_example
-   getting_started/run_on_flyte
-   getting_started/learn
-
-.. toctree::
-   :caption: How-Tos
-   :maxdepth: 1
-   :name: howtotoc
-
-   plugins/index
-   howto/index
-
-.. toctree::
-   :caption: Deep Dive
-   :maxdepth: 1
-   :name: divedeeptoc
-
-   dive_deep/index
-   reference/index
-
-.. toctree::
-   :caption: Contributor Guide
-   :maxdepth: 1
-   :name: roadmaptoc
-
-   community/index
-   community/docs
-   community/roadmap
-   community/compare
+Meet Flyte
+==========
+
+Flyte is an open-source, container-native, structured programming and distributed processing platform. It enables highly concurrent, scalable and maintainable workflows for machine learning and data processing.
+
+Created at Lyft, Flyte provides first class support for Python, and is built directly on Kubernetes for all the benefits containerization provides: portability, scalability, and reliability.
+
+Flyte provides a single unit of execution (task) as a top-level concept. Multiple tasks arranged in a data producer-consumer order create the workflow, which is pure specification created in any language.
+
+Why Flyte?
+==========
+
+Flyte's main purpose is to increase the development velocity for data processing and machine learning, enabling large-scale compute execution without the operational overhead. Teams can therefore focus on the business goals and not the infrastructure.
+
+What makes Flyte different?
+---------------------------
+
+* Fully Containerized
+* Extensible Backend & SDK’s
+* Ergonomic SDK’s in Python and Java/Scala
+* Versioned & Auditable - all actions are recorded
+* Horizontally Scalable - execute a single task or a workflow, attach multiple schedules to a workflow
+* Battle-tested - millions of pipelines executed per month
+* Vertically-Integrated Compute - serverless experience
+* Deep understanding of data-lineage & provenance
+* Operation Visibility - cost, performance, etc.
+* Cross-Cloud Portable Pipelines
+
```

Comment:
Other points that you could add:
* Memoization
* Dynamic Workflows

Reply:
Done.